Library/Formula/log4cxx.rb

```diff
@@ -37,25 +39,137 @@ def install
   end
 end
 __END__
+diff --git a/src/main/cpp/locationinfo.cpp b/src/main/cpp/locationinfo.cpp
+index e76ea29..2e1950c 100644
+--- a/src/main/cpp/locationinfo.cpp
++++ b/src/main/cpp/locationinfo.cpp
```

Comment:
Will do. I also forgot to do the same with cryptopp, will do an update soon.

As for patch to log4cxx project.  The issue tracking for log4cxx project is not working right now, but given that the project has not seen a new release since 2008, I would not expect much...